src/strokes.ts

```diff
@@ -233,15 +237,15 @@ export class Stroke extends Modifier {
     }
 
     // Draw the arrow head
-    Glyph.renderGlyph(ctx, x + this.x_shift + arrow_shift_x, arrow_y, this.render_options.font_scale, arrow, {
-      category: `stroke_${strokeLine}.${arrow}`,
+    Glyph.renderGlyph(ctx, x + this.xShift + arrowShiftX, arrowY, this.renderOptions.fontScale, arrow, {
+      category: `stroke${type}.${arrow}`,
```

Comment:
This could use the proposed `camelCase` utility function.

Reply:
Not necessary here. The first character is already capitalized since it is a hard coded string.